.buildkite/pipeline.yml

```diff
@@ -0,0 +1,47 @@
+# Nodes with values to reuse in the pipeline.
+common_params:
+  plugins: &common_plugins
+  - &bash_cache automattic/bash-cache#v1.3.2: ~
+  # Common environment values to use with the `env` key.
+  env: &common_env
+    IMAGE_ID: xcode-12.5.1
```

Comment:
This _could_ become a root element, too, as per [this](https://github.com/Automattic/ScreenObject/pull/6/files#r695336348).

```yml
env:
  IMAGE_ID: xcode-12.5.1
```

Reply:
Except if not all steps need that env… and I think they don't all do (see comment below) 🤔